examples/workflow_example_cep_plot.jl

```diff
@@ -4,25 +4,29 @@ using Clp
 using Plots
 
 ## LOAD DATA ##
-state="GER_1" # or "GER_18" or "CA_1" or "TX_1"
+state="GER_18" # or "GER_18" or "CA_1" or "TX_1"
 # laod ts-data
-ts_input_data = load_timeseries_data("CEP", state; T=24) #CEP
+ts_input_data = load_timeseries_data_provided(state; T=24) #CEP
 # load cep-data
 cep_data = load_cep_data_provided(state)
 
 ## CLUSTERING ##
 # run aggregation with kmeans
-ts_clust_data = run_clust(ts_input_data;method="kmeans",representation="centroid",n_init=5,n_clust=5) # default k-means make sure that n_init is high enough otherwise the results could be crap and drive you crazy
+ts_clust_data = run_clust(ts_input_data;method="kmeans",representation="centroid",n_init=10000,n_clust=5) # default k-means make sure that n_init is high enough otherwise the results could be crap and drive you crazy
```

Comment:
How about keeping it at 100 or 1000 and mentioning in a comment that 10000 should be used for real applications?

Reply:
Yeah